src/Pages/ProductScreen/ProductPage.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react'
+import Pagination from '../../Components/Pagination/Pagination'
+import ProductList from '../../Components/ProductList/ProductList'
+import SideBar from '../../Components/SideBar/SideBar'
+import { useLoading, ThreeDots } from '@agney/react-loading';
+import './ProductPage.css'
+
+function ProductPage() {
+
+    const [loading, setLoading] = useState(true);
```

Comment:
Use is, at the start of boolean variable names.

Reply:
Fixed!!